chart/kubeapps/templates/kubeappsapis/rbac_fluxv2.yaml

```diff
@@ -0,0 +1,47 @@
+{{- if .Values.featureFlags.kubeappsAPIsServer }}
```

Comment:
I'll do a separate PR to remove that option... it should no longer be an option since the UI is using it now.

Reply:
sounds good